specifyweb/frontend/js_src/css/workbench.css

```
@@ -28,14 +28,38 @@
 .handsontable .colHeader.columnSorting::before {
-  @apply !static order-2 !mt-1 !bg-center pr-1;
+  @apply !absolute !top-1/2 !bg-center flex-none;
+  
+  right: 0.375rem;
+  transform: translateY(-50%);
+  margin-inline-end: 0;
+}
+
+.handsontable .columnSorting.columnSorting.columnSorting.sortAction.ascending,
+.handsontable .columnSorting.columnSorting.columnSorting.sortAction.descending {
+  /* override handsontable's large padding when sorting direction is present*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)